Javascript/Housey pricey/house.js

```diff
@@ -0,0 +1,37 @@
+// Peter's parameters
+let peterWidth = 8;
+let peterHeight = 10;
+let peterDepth = 10;
+let peterGardenSize = 100;
+let peterRealPrice = 2500000;
+
+// Julia's parameters
+let juliaWidth = 5;
+let juliaHeigth = 8;
+let juliaDepth = 11;
+let juliaGardenSize = 70;
```

Comment:
Well done using [`cammelCase`](https://www.freecodecamp.org/news/snake-case-vs-camel-case-vs-pascal-case-vs-kebab-case-whats-the-difference/#camel-case)

Reply:
***Thank you!****